src/prefect/schedules/filters.py

```diff
@@ -0,0 +1,111 @@
+"""
+Schedule filters are functions that accept a candidate `datetime` and return `True` if
+the candidate is valid, and `False` otherwise.
+
+Filters have the signature `Callable[[datetime], bool]`.
```

Comment:
Is there a plan to support custom filters? I understand these get us most of the way there but what if I want to build a NYSE holiday filter to reuse.

Reply:
Custom filters are supported in Core, but to work in Cloud schedules must be safely serializable. Functions must be added to Core (request for PR ;))